core-util/FunctionPointer.h

```diff
@@ -83,10 +83,14 @@ class FunctionPointer0 : public FunctionPointerBase<R>{
     R call(){
         return FunctionPointerBase<R>::call(NULL);
     }
+    R operator ()(void) {
+        return FunctionPointerBase<R>::call(NULL);
+    }
 
     FunctionPointerBind<R> bind() {
-        FunctionPointerBind<R> fp;
-        fp.bind(&FunctionPointerBase<R>::_nullops, (ArgStruct *) NULL, this);
+        FunctionPointerBind<R> fp(*this);
+        void * storage = this->preBind(fp, (ArgStruct *)NULL, &FunctionPointerBase<R>::_nullops);
```

Comment:
We've been though this. Yes, MINAR is the pretty much the only part which is camelCase, the vast majority of mbed OS is not. Let's stick to our regular coding style, we'll fix MINAR later.